lib/csv/table.rb

```diff
@@ -23,8 +23,19 @@ class Table
     # * length()
     # * size()
     #
-    def initialize(array_of_rows)
-      @table = array_of_rows
+    def initialize(array_of_rows, headers=nil)
+      if headers.nil?
```

Comment:
This is only style, and also opinion: can we invert this to use ruby's truthy?
```ruby
if headers
  @headers = headers
else
  if @table.empty?
   # ...
  end
end
```

Reply:
That it is clear, I've fixed it.